### PR TITLE
Release Akka 2.5.18

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: blog/_posts/2018-10-07-akka-2.5.18-released.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+* Async DNS over TCP in case of truncated UDP responses [#25460](https://github.com/akka/akka/issues/25460)
+* Fix DNS SRV records with port numbers larger than 32767 [#25760](https://github.com/akka/akka/issues/25760)
+* Fix DNS cache time unit [#25860](https://github.com/akka/akka/issues/25860)
+* Support IPv6 in Async DNS [#25687](https://github.com/akka/akka/pull/25687)
+* Fix Java APIs when running with JDK 11 [#25777](https://github.com/akka/akka/issues/25777)
+* New `MemberDowned` cluster event [#25854](https://github.com/akka/akka/pull/25854)
+* More examples of Streams operators, thanks to [@salehkhazaei](https://github.com/salehkhazaei) and [@mehdimousavi1995](https://github.com/mehdimousavi1995). More contributions are welcome, see [#25468](https://github.com/akka/akka/issues/25468)
+* Update to Aeron 1.11 [#25743](https://github.com/akka/akka/pull/25743)
+
+**Unfortunately Akka 2.5.18 doesn't work in Eclipse.** We will release 2.5.19 as soon as this regression has been resolved.
```
Comment (raboof) [inline blog/_posts/2018-10-07-akka-2.5.18-released.md]: That's weird, is it something fishy we're doing or some limitation of Eclipse?

File: blog/_posts/2018-10-07-akka-2.5.18-released.md [hunk 2]
```diff
@@ -20,7 +20,10 @@ We are pleased to announce a new patch release of Akka 2.5. Some notable improve
 * More examples of Streams operators, thanks to [@salehkhazaei](https://github.com/salehkhazaei) and [@mehdimousavi1995](https://github.com/mehdimousavi1995). More contributions are welcome, see [#25468](https://github.com/akka/akka/issues/25468)
 * Update to Aeron 1.11 [#25743](https://github.com/akka/akka/pull/25743)
 
-A total of 66 issues were closed since 2.5.17. The complete list can be found on the [2.5.18 milestone](https://github.com/akka/akka/milestone/134?closed=1) on github.
+**Unfortunately Akka 2.5.18 doesn't work in Eclipse.** We will release 2.5.19 as soon as this regression has been resolved.
+If you are using Eclipse we recommend that you stay on Akka version 2.5.17.
+
+A total of 6 issues were closed since 2.5.17. The complete list can be found on the [2.5.18 milestone](https://github.com/akka/akka/milestone/134?closed=1) on github.
```
Comment (dwijnand) [inline blog/_posts/2018-10-07-akka-2.5.18-released.md]: 66 issues

File: blog/_posts/2018-10-07-akka-2.5.18-released.md [hunk 3]
```diff
@@ -20,7 +20,10 @@ We are pleased to announce a new patch release of Akka 2.5. Some notable improve
 * More examples of Streams operators, thanks to [@salehkhazaei](https://github.com/salehkhazaei) and [@mehdimousavi1995](https://github.com/mehdimousavi1995). More contributions are welcome, see [#25468](https://github.com/akka/akka/issues/25468)
 * Update to Aeron 1.11 [#25743](https://github.com/akka/akka/pull/25743)
 
-A total of 66 issues were closed since 2.5.17. The complete list can be found on the [2.5.18 milestone](https://github.com/akka/akka/milestone/134?closed=1) on github.
+**Unfortunately Akka 2.5.18 doesn't work in Eclipse.** We will release 2.5.19 as soon as this regression has been resolved.
```
Comment (dwijnand) [inline blog/_posts/2018-10-07-akka-2.5.18-released.md]: Provide a tracking issue link?

**Comment by dwijnand** [comment]
Mention `PersistentBehavior.withEnforcedReplies`? https://github.com/akka/akka/issues/25482

File: blog/_posts/2018-10-07-akka-2.5.18-released.md (hunk 2 above)
Comment (patriknw) [reply blog/_posts/2018-10-07-akka-2.5.18-released.md]: :-) thanks

**Comment by patriknw** [comment]
> Mention PersistentBehavior.withEnforcedReplies

It's not done yet. Java API is missing.

**Pull request merged**